Makefile

```diff
@@ -165,7 +165,7 @@ lintfix: ## Applies the lint changes.
 .PHONY: release
 release: generate ## Upload artifacts to Sonatype Nexus.
 	./gradlew $(GRADLE_ARGS) --info publish --stacktrace --no-daemon --no-parallel
-	./gradlew $(GRADLE_ARGS) --info closeAndReleaseRepository
+	./gradlew $(GRADLE_ARGS) --info releaseRepository
```

Comment:
Missed this from release notes of gradle plugin: https://github.com/connectrpc/connect-kotlin/pull/241